action.py

```diff
@@ -37,7 +37,8 @@
 _SUMMARY = Path(_summary_path).open("a")
 
 _RENDER_SUMMARY = os.getenv("GHA_SIGSTORE_PYTHON_SUMMARY", "true") == "true"
-_DEBUG = os.getenv("GHA_SIGSTORE_PYTHON_INTERNAL_BE_CAREFUL_DEBUG", "false") != "false"
+_INTERNAL_DEBUG = os.getenv("GHA_SIGSTORE_PYTHON_INTERNAL_BE_CAREFUL_DEBUG", "false") != "false"
+_GHA_STEP_DEBUG = os.getenv("ACTIONS_STEP_DEBUG", "false") == "true"
```

Comment:
Let's just merge these two: `ACTIONS_STEP_DEBUG` should be a fallback value for the original `_DEBUG` variable. 

(This also avoids the need for two different `print(...)` calls below.)

Reply:
The old one, since `::debug` would only be visible when debugging is enabled through the runner setting rather than the action's own input.